### DISCUSSION TITLE
Where to Find the Project and Task Template IDs

### DISCUSSION THREAD

**Discussion by srirag21** [semaphoreui/semaphore#2104] [Q&A] [answered]
I currently have an instance of Semaphore launched however in order to use many of the API capabilities, I have to have access to a particular project ID and task template ID however I cannot find these anywhere.

**Comment by fiftin** [comment, marked as answer]
Hi @srirag21 
only in URL
![image](https://github.com/semaphoreui/semaphore/assets/914224/9e301006-9a0d-4b67-bcc8-eb808928c8eb)
